### PR TITLE
[#76] SOD offset now includes the dates timestamp

### PR DESCRIPTION
**Pull request by chimano**
- last_sod_offset now returned to the frontend when requesting the orderbook
- index created to optimize sorting of messages by sod_offset

### CODE REVIEW

File: core/db/db_setup.js [hunk 1]
```diff
@@ -5,5 +5,7 @@ db.createCollection('orderbooks');
 db.createCollection('messages');
 
 db.orderbooks.createIndex({ instrument: 1, timestamp: 1 });
+
 db.messages.createIndex({ instrument: 1, timestamp: 1 });
 db.messages.createIndex({ instrument: 1, sod_offset: 1 });
+db.messages.createIndex({ instrument: 1, sod_offset: -1 });
```
Comment (alvyn279) [inline core/db/db_setup.js]: Why is this necessary?

Comment (chimano) [reply core/db/db_setup.js]: We have to sort by sod_offset in both ways for pagination. Direction of the index matters for composite keys in mongodb. https://stackoverflow.com/questions/10329104/why-does-direction-of-index-matter-in-mongodb
